themes/inbeat/layouts/partials/site-header.html

```diff
@@ -152,7 +152,9 @@ <h2>{{ .topText }}</h2>
 			</div>
 		</div>
 	</div>
-	{{$url := print "/articles/" .File.BaseFileName "/"}} {{ if and (eq .Page.RelPermalink $url) }}
+  {{ if .File }}
+    {{$url := print "/articles/" .File.BaseFileName "/"}} {{ if and (eq .Page.RelPermalink $url) }}
```

Comment:
Are we sure this works? `$url` would be available just in the loop no? And then what if there's no `.File`?

You could define an empty value for the whole file and then reassign if there's a `.File`

Reply:
I'm not sure it really only happen on my mac.. For now Ill remove it and just add it on local when I need to make hugo work